configure.ac

```diff
@@ -61,6 +61,7 @@ AC_ARG_ENABLE([debug],
                              [enable debugging output])],[],[enable_debug=yes])
 AM_CONDITIONAL([ENABLE_DEBUG], test x"$enable_debug" = x"yes")
 if test x"$enable_debug" = x"yes"; then
+   CFLAGS="${CFLAGS} -O0 -g3 -fsanitize=address -fno-omit-frame-pointer"
```

Comment:
I think it's mainly because I specified -O0. I can increase the optimization level.